frontend/helpers/generateFakeCollateral.js

```diff
@@ -4,13 +4,18 @@ import COLLATERALS from 'auctions-core/src/constants/COLLATERALS';
 
 export const generateFakeCollaterals = function () {
     const collaterals = [];
```

Comment:
> Please don't forget to update corresponding type

```suggestion
    const collaterals: CollateralRow[] = [];
```

This will help enforcing the correct type